core/frontend/src/components/scanner/servicesScanner.vue

```diff
@@ -0,0 +1,59 @@
+<template>
+  <span />
+</template>
+
+<script lang="ts">
+import Vue from 'vue'
+import axios from 'axios'
+import { getModule } from 'vuex-module-decorators'
+import ServicesScannerStore from '@/store/servicesScanner'
+import {Service} from '@/types/SERVICE'
+
+const servicesHelper: ServicesScannerStore = getModule(ServicesScannerStore)
+
+/**
+ * Actual scanner for running services.
+ * This periodically fetches /helper/latest/web_services
+ * and updates the ServiceHelperStore with the available services
+ * @displayName Services Scanner
+ */
+export default Vue.extend({
+  name: 'ServicesFetcher',
+
+  mounted () {
+    // Fetch network data
+    this.requestData()
+
+    // Fetch periodic API request
+    this.startPeriodicRequest()
+  },
+  methods: {
+    startPeriodicRequest () {
+      setInterval(() => {
```

Comment:
Just to make sure, if we mount, by mistake, this module more than one time, will this create two periodic requests ?
I'm saying that since the return variable for the `setInterval` is not being used and the RAII will not run (?).
Maybe we should add a variable for it and delete and delete in the destructor, but I'm not sure how typescript will deal with the actual scenario.

Reply:
good catch!